lunatrace/bsl/hasura/migrations/lunatrace/1668718323683_add_package_manager_to_manifest/up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE public.resolved_manifest ADD COLUMN package_manager TEXT NULL;
+ALTER TABLE public.resolved_manifest ADD COLUMN lockfile_version INT NULL;
+
+COMMENT ON TABLE public.manifests IS 'DEPRECATED DEPRECATED DEPRECATED !!!!!!!';
```

Comment:
nit: "Deprecated. Use ..."

Reply:
Loud is good for these. Professionalism is way less important than me remembering this is not the right table when I see it in hasura.

Reply:
I'd agree with chris on this one. This is exposed all the way through the graphql schema and codegen'd clients.

Reply:
Hm ok